[MKC-718] IoTC Maintenance (#607)
arduino/docs-content: content/arduino-cloud/01.getting-started/06.dashboard-widgets/dashboard-widgets.md

Serial.println(stepperVariable);
```

### Time Picker

![Time Picker Widget](assets/time-picker.png)

> The time picker widget is used to retrieve the local time on your board.

Reviewer: ```suggestion
The time picker widget is used to represent and set a time and/or a date.
```

Reviewer: It represents whatever time the variable contains, then it's up to you what time you want to put in the variable :)

> Can be linked with the **CloudTime** variable.

Reviewer: ```suggestion
Can be linked with a **CloudTime** variable.
```

An example of how it is used in a sketch:

```
CloudTimeVariable = ArduinoCloud.getLocalTime();
```

> ***`ArduinoCloud.getLocalTime()` returns the Unix Time Stamp (counted in seconds).***

Reviewer: ```suggestion
In this example, the variable named `CloudTimeVariable` is populated with `ArduinoCloud.getLocalTime()` which returns the Unix time stamp (in seconds) of the local time zone.
```
